Mathlib/LinearAlgebra/QuadraticForm/TensorProduct.lean

```diff
@@ -52,14 +85,15 @@ noncomputable def tensorDistrib :
       (QuadraticMap.associated : QuadraticForm A M₁ →ₗ[A] BilinForm A M₁)
       (QuadraticMap.associated : QuadraticForm R M₂ →ₗ[R] BilinForm R M₂)
   toQ ∘ₗ tmulB ∘ₗ toB
+--
```

Comment:
What's this?

Reply:
Must be an artifact. Removed now, thanks.